Sources/App/Settings/SettingsDetailViewController.swift

```diff
@@ -951,4 +966,8 @@ enum OpenInBrowser: String, CaseIterable {
             return true
         }
     }
+
+    var supportsPrivateTabs: Bool {
+        [.Firefox].contains(self)
```

Comment:
this is fine, though in general you should prefer a `switch` for something like this as it forces you to consider new cases when adding them

Reply:
Absolutely, makes sense. Changing it to a switch will also allow for more elaborate checks, more than just a hardcoded true/false.